scripts/run_server.sh

```diff
@@ -8,4 +8,4 @@ rm -rf ./node_modules
 yarn install
 
 # Running server
-yarn start
\ No newline at end of file
+yarn run serve
```

Comment:
Is this for the deployment on heroku or just a script we can use to run it locally?

Reply:
This is for Heroku to serve from the dist folder